girder/__init__.py

```diff
@@ -259,6 +260,26 @@ def logprint(*args, **kwargs):
         _originalStdOut.flush()
 
 
+def _setupCache():
+    """
+    Setup caching based on configuration file.
+
+    If the cache is disabled, both the global and per-request cache are set
+    to the null backend, effectively making them a noop. This allows for
+    decorating functions with the cache region, despite it being unused.
+    """
+    curConfig = config.getConfig()
+
+    if not curConfig['cache']['enabled']:
+        cache.configure(backend='dogpile.cache.null')
+        requestCache.configure(backend='dogpile.cache.null')
+    else:
+        cache.configure_from_config(curConfig['cache'], 'cache.global.')
```

Comment:
Is there reason to have the word `cache` in the setting name? Seems redundant with it being in the `cache` section.